packages/content/src/requests/items/registerAttributeListener/RegisterAttributeListenerAcceptResponseItem.ts

```diff
@@ -17,7 +17,7 @@ export class RegisterAttributeListenerAcceptResponseItem extends AcceptResponseI
     public listenerId: string;
 
     public static override from(
-        value: IRegisterAttributeListenerAcceptResponseItem | RegisterAttributeListenerAcceptResponseItemJSON
+        value: IRegisterAttributeListenerAcceptResponseItem | Omit<RegisterAttributeListenerAcceptResponseItemJSON, "@type">
```

Comment:
you should still allow to add `@type`, so `IX | Omit<XJSON, "@type"> | XJSON` would be better here